lib/components/form/batch-settings.tsx

```diff
@@ -81,6 +82,7 @@ function BatchSettings({
   modeSettingValues,
   onPlanTripClick,
   routingQuery,
+  setQueryParam,
   setUrlSearch,
```

Comment:
Do we still need `setUrlSearch`? Is it used somewhere else? Should we use something else instead?